lib/ansible/cli/arguments/option_helpers.py

```diff
@@ -178,13 +178,14 @@ def version(prog=None):
 # Functions to add pre-canned options to an OptionParser
 #
 
-def create_base_parser(usage="", desc=None, epilog=None):
+def create_base_parser(usage="", desc=None, epilog=None, prog=None):
```

Comment:
@abadger I'm not sure in `prog=None` is a good default here.

Reply:
Yeah, prog isn't really optional anymore.  I'd probably change the function signature to:
``` python
def create_base_parser(prog, usage="", desc=None, epilog=None):
```

and update the places in ```lib/ansible/cli/*``` that call the function.

Reply:
Done.